.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Feature request
+about: Suggest a new idea for the package.
+title: '💡 '
+labels: ''
+assignees: ''
+
+---
+
+## Use case
+
+<!--
+     Please tell us the problem you are running into that led to you wanting
```

Comment:
Not happy with how this is worded, not every new feature request might be because of a problem